server/controllers/stock/core.js

```diff
@@ -402,14 +408,38 @@ function stockManagementProcess(inventories) {
     Q = inventory.quantity; // the quantity
     CM = inventory.avg_consumption; // consommation mensuelle
     CM_NOT_ZERO = !CM ? 1 : CM;
+
```

Comment:
Thanks for adding these comments!   I was looking at these lines yesterday and thought that we should beef up the comments!   Great addition!